pluma/pluma-window.c

```diff
@@ -77,214 +77,214 @@ static gboolean cansave = TRUE;
 /* Signals */
 enum
 {
-	TAB_ADDED,
-	TAB_REMOVED,
-	TABS_REORDERED,
-	ACTIVE_TAB_CHANGED,
-	ACTIVE_TAB_STATE_CHANGED,
-	LAST_SIGNAL
+    TAB_ADDED,
+    TAB_REMOVED,
+    TABS_REORDERED,
+    ACTIVE_TAB_CHANGED,
+    ACTIVE_TAB_STATE_CHANGED,
+    LAST_SIGNAL
 };
 
 static guint signals[LAST_SIGNAL] = { 0 };
 
 enum
 {
-	PROP_0,
-	PROP_STATE
+    PROP_0,
+    PROP_STATE
 };
 
 enum
 {
-	TARGET_URI_LIST = 100
+    TARGET_URI_LIST = 100
 };
 
 G_DEFINE_TYPE_WITH_PRIVATE (PlumaWindow, pluma_window, GTK_TYPE_WINDOW)
 
-static void	recent_manager_changed	(GtkRecentManager *manager,
-					 PlumaWindow *window);
+static void    recent_manager_changed    (GtkRecentManager *manager,
+                                          PlumaWindow *window);
```

Comment:
the names of the arguments are not aligned in the function header, there are more matches in this file.